Userland/Libraries/LibWeb/HTML/Window.h

```diff
@@ -92,6 +92,12 @@ class Window final
 
     ImportMap const& import_map() const { return m_import_map; }
 
+    void set_import_map(ImportMap& import_map)
+    {
+        // FIXME: Assert that the existing import_map is empty
+        m_import_map = import_map;
+    }
```

Comment:
"Register an import map" is a [spec algorithm](https://html.spec.whatwg.org/multipage/webappapis.html#register-an-import-map). The way we do this is:
1. Name the function/method the same as the spec algorithm. (`register_an_import_map(...)`)
2. Add a comment with a link to the spec.
3. Include all the spec steps as comments. Put `FIXME` before the ones that aren't implemented.

The reasons for this are, it makes it clearer that the code is trying to follow the spec (as some older code is ad-hoc), it's more obvious when something is wrong, and it's easier to tell if the spec has changed since we implemented the code.

As a side note, this should probably be implemented in Window.cpp